Homework_30/func.js

```diff
@@ -0,0 +1,60 @@
+//1
+let a = new Array(13, 22, 31, 4, 5, 16, 7, 8, 9, 10);
+let isPassed = true;
+//2
+let functions = {
+    checkArray: function (a, isPassed) {
+        let sum = 0;
+        for (let i = 0; i < a.length; i++) {
+            if (typeof a[i] === "number") {
+                sum += a[i];
+            } else {
+                alert("There's not a number in the array");
+                isPassed = false;
+                break;
+            }
+        }
+        if (isPassed === true) {
```

Comment:
why is this param `isPassed` is needed?
the main rule of writing unit-tests: you should never change your implementation only for testing reasons